source/reference/cutover-process.txt

```diff
@@ -10,13 +10,21 @@ Cutover Process
    :local:
    :backlinks: none
    :class: singlecol
+   :depth: 1
 
-You can finalize a migration and transfer your application's writes from
+You can finalize a migration and transfer your application load from
 the source to the destination cluster using the ``mongosync`` cutover
 process. 
 
-Procedure 
----------
+.. note::
+
+   Before you cut your application over to the destination
```

Comment:
I'd rewrite or add something to the start of the page. "Cut" has multiple meanings and is not global. In this case it means "migrate and transfer" or similar. Consider rewrite or clarify. Cheers.

Reply:
Fixed